tests/test_training.py

```diff
@@ -50,9 +51,10 @@ def _process_batch(
 class NaNTrainingLoop(SLCWATrainingLoop):
     """A wrapper around SLCWATrainingLoop returning NaN losses."""
 
-    def __init__(self, model: Model, patience: int):
+    def __init__(self, model: Model, patience: int, automatic_memory_optimization: bool = False):
         super().__init__(model=model, optimizer=optim.Adam(lr=1.0, params=model.parameters()))
         self.patience = patience
+        self.automatic_memory_optimization = automatic_memory_optimization
```

Comment:
should be passed via `super().__init__`

Reply:
Done in de0d8987908ce329a4a398474f84690dcf5d4f5d.